wormhole-connect/src/hooks/useBalanceChecker.ts

```diff
@@ -0,0 +1,61 @@
+import { useEffect, useState, useMemo } from 'react';
+import { amount, routes } from '@wormhole-foundation/sdk';
+import { getWalletConnection, TransferWallet } from 'utils/wallet';
+import config from 'config';
+import { useIsMounted } from './useIsMounted';
+
+export const useBalanceChecker = (
+  quote?: routes.Quote<
+    routes.Options,
+    routes.ValidatedTransferParams<routes.Options>
+  >,
+): {
+  feeSymbol?: string;
```

Comment:
Aren't these return props very similar to `WalletBalanceWarningProps` below? Can you import it here as well? Or maybe move it to `wormhole-connect/src/config/types.ts`?